doc/getting-started/rt_industry_ubuntu.rst

```diff
@@ -155,6 +155,20 @@ Build the ACRN Hypervisor on Ubuntu
 
       $ sudo pip3 install kconfiglib
 
+#. Install IASL in Ubuntu for User VM launch
+   ACRN uses iasl to parse User VM ACPI information.
+   The original iasl in Ubuntu 18.04 is too old; update it using the following steps:
+
+   .. code-block:: none
+
+   $ sudo -E apt-get install iasl
```

Comment:
Why installing `iasl` if we know it's too old. Let's get rid of this altogether.

Reply:
The paragraph is misaligned, it will not render correctly. Each line needs to start aligned with the `c` from `code-block`